src/odemis/acq/stream/_static.py

```diff
@@ -422,12 +422,12 @@ def dis_bbtl(v):
 
         # polarization VA
         # check if any polarization analyzer data, (None) == no analyzer data (pol)
-        if self._pos.keys()[0][-1]:
+        if list(self._pos.keys())[0][-1]:
```

Comment:
This one is a little annoying, as _pos might be a little big. Anyway, it should be possible to just replace by `polpositions != {None}` and in the next line `list(polpositions)[0]`.